modules/nf-core/checkqc/main.nf

```diff
@@ -0,0 +1,47 @@
+process CHECKQC {
+    label 'process_single'
+
+    conda "bioconda::checkqc=3.6.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/checkqc:3.6.1--py_0':
+        'biocontainers/checkqc:3.6.1--py_0' }"
+
+    input:
+    path(run_dir)
+    path(checkqc_config)
+
+    output:
+    path "*checkqc_report.json", emit: report
+    path "versions.yml"        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def config = checkqc_config ? "--config $checkqc_config" : ''
+
+    """
+    checkqc \
+        $args \
+        $config \
+        --json \
+        $run_dir > checkqc_report.json || test -f "checkqc_report.json"
```

Comment:
```suggestion
        $run_dir > checkqc_report.json || test -s "checkqc_report.json"
```
Sorry, one more suggestion. Test the file is not size 0, rather than it's a regular file as it will be created as one.

Reply:
Ah smart! I was just thinking that one would like to make sure it's not empty :+1: